src/main/resources/fabric.mod.json

```diff
@@ -10,6 +10,7 @@
   ],
   "contact": {
     "homepage": "https://www.curseforge.com/minecraft/mc-mods/inventory-tabs",
+    "sources": "https://github.com/cakewhip/inventorytabs/"
```

Comment:
Comma? I'd be spooked if this worked without the comma

Reply:
https://github.com/cakewhip/inventorytabs/pull/8/commits/fd1cb389600f8259f902fdbf24a362dc177355ab